d2bs/kolbot/libs/common/Prototypes.js

```diff
@@ -1104,3 +1104,145 @@ if (!Array.prototype.find) {
 		writable: true
 	});
 }
+
+/**
+ * @description Return the first element or undefined
+ */
```

Comment:
add @returns?

Reply:
it is unknown what is returned. Since it returns the first element, which can be everything.

However, i added a @return undefined|*